spec/models/dish_spec.rb

```diff
@@ -1,6 +1,9 @@
 require 'rails_helper'
 
 RSpec.describe Dish, type: :model do
```

Comment:
Don't forget to test your model associations as well as your instance methods! No need to add or test validations, but we do want you to add the shoulda matchers tests for relationships